config/boards/ayn-odin2.conf

```diff
@@ -0,0 +1,87 @@
+# Generate kernel and rootfs image for Qcom ABL booting
+declare -g BOARD_NAME="Ayn Odin2"
+declare -g BOARD_MAINTAINER="FantasyGmm"
+declare -g BOARDFAMILY="qcom-abl"
+declare -g KERNEL_TARGET="sm8550"
+declare -g KERNELPATCHDIR="arm64-sm8550"
+declare -g EXTRAWIFI="no"
+declare -g BOOTCONFIG="none"
+declare -g IMAGE_PARTITION_TABLE="gpt"
+declare -g -a ABL_DTB_LIST=("qcs8550-ayn-odin2-hypdtbo")
+
+# Use the full firmware, complete linux-firmware plus Armbian's
+declare -g BOARD_FIRMWARE_INSTALL="-full"
+
+function post_family_config_branch_sm8550__pmos_kernel() {	
```

Comment:
This is kernel is no longer from pmos, so it's better to user another name.

Reply:
ok,i will fix this